apps/mymedicare_cb/views.py

```diff
@@ -61,6 +61,11 @@ def authenticate(request):
     # Get the userinfo response object
     user_info = response.json()
     user = get_and_update_user(user_info)
+    logger.info({
+        "sub": user_info["sub"],
+        "username": user.username,
+        "user_id_hash": user.crosswalk.user_id_hash,
+        "fhir_id": user.crosswalk.fhir_id})
```

Comment:
Can/should we log user.id and crosswalk.id here?

Reply:
I agree with Tisza.